src/libraries/System.Threading.Channels/src/System.Threading.Channels.csproj

```diff
@@ -17,7 +17,7 @@
     <Compile Include="System\Threading\Channels\ChannelClosedException.netcoreapp.cs" Condition="!$(TargetFramework.StartsWith('netstandard'))" />
     <Compile Include="System\Threading\Channels\ChannelOptions.cs" />
     <Compile Include="System\Threading\Channels\ChannelReader.cs" />
-    <Compile Include="System\Threading\Channels\ChannelReader.netcoreapp.cs" Condition="!$(TargetFramework.StartsWith('netstandard'))" />
+    <Compile Include="System\Threading\Channels\ChannelReader.netcoreapp.cs" Condition="!$(TargetFramework.StartsWith('netstandard')) OR '$(TargetFramework)' == 'netstandard2.1'" />
```

Comment:
You can rename the file to .netstandard21.cs

Reply:
done